src/Domain/Creators/BinaryCompactObject.hpp

```diff
@@ -771,7 +721,19 @@ class BinaryCompactObject : public DomainCreator<3> {
   std::array<double, 2> initial_size_map_accelerations_{
       std::numeric_limits<double>::signaling_NaN(),
       std::numeric_limits<double>::signaling_NaN()};
-  std::array<std::string, 2> size_map_function_of_time_names_;
+  std::array<std::string, 2> size_map_function_of_time_names_{
+      {"SizeA", "SizeB"}};
+  // mutable because the expiration times can potentially be changed in the
+  // const member `functions_of_time` if there are control systems
```

Comment:
This does not seem like a good use of `mutable`.  If this is observable state it shouldn't be changed by a const method.  If it is not observable state do you need the variable?

Reply:
I believe the domain creator only exists during option parsing, so it doesn't seem like a place to store any state